app/src/main/java/ro/code4/monitorizarevot/helper/WebClient.kt

```diff
@@ -1,10 +1,10 @@
 package ro.code4.monitorizarevot.helper
 
 import android.graphics.Bitmap
+import android.net.Uri
 import android.webkit.WebResourceRequest
 import android.webkit.WebView
 import android.webkit.WebViewClient
-import ro.code4.monitorizarevot.BuildConfig
 
 class WebClient(private val listener: WebLoaderListener) : WebViewClient() {
```

Comment:
I hope this class will be used more than in one fragment. Otherwise, I want to merge it as an anonymous class into `GuideFragment`

Reply:
I don't think it will be needed in any more fragments any time soon. You are right, it would be better be placed in `GuideFragment`

Reply:
Ok, I'll merge it :)